Attempt to change the reversal section into something more constructive

rubinovitz opened the pull request:
An attempt to allow more open discussion about the reversal rules

rubinovitz committed:
Attempt to change the reversal section into something more constructive

kern commented:
I like the tone change and reframe. The new paragraph reads well, but I'm not sure if the last sentence is necessary. Sounds a little patronizing.

rubinovitz commented:
@kern  that's fair. I'm trying to emphasize we care about everyone, but if that sentence does more harm than good, happy to remove it.

rubinovitz committed:
Shorten reversal section for clairty

rubinovitz committed:
Offer to open a dialog about reversals

This could lead to more work for moderators but we can figure out the best way to educate people on these issues and better ways to communicate in this document if its unclear.
